app/controllers/meals_controller.rb

```diff
@@ -7,8 +7,9 @@ class MealsController < ApplicationController
   def index
     @meals = current_user.meals.includes(:recipe)
     @meals_json = @meals
+      .sort_by(&:time_position)
       .collect{ |meal| {
-      title: meal.recipe.name,
+      title: "(#{meal.time.humanize[0]}) #{meal.recipe.name}",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.